UniversalDashboard.Forge.psm1

```diff
@@ -63,15 +63,15 @@ function New-UDDesktopApp {
         if ($PathInfo.PSIsContainer)
         {
             Write-Verbose "Path is a directory. Locating dashboard.ps1"
-            $Dashboard = Join-Path $PathInfo.FullName "dashboard.ps1"
-            if (-not (Test-Path $Dashboard))
+            $dashboard = Join-Path $PathInfo.FullName "dashboard.ps1"
```

Comment:
Changed variables that are not parameters to camelcase throughout all code to comply with Powershell best practices.